docs/source/dataset_library_api.rst

```diff
@@ -0,0 +1,15 @@
+*******************
+Dataset Library API
```

Comment:
Could you call this simply `Dataset Library`. The "API" bit in the file naming is just to distinguish from the dataset_library directory where we store the csv files.